compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/test/decompose_online_attention.mlir

```diff
@@ -165,3 +167,67 @@ func.func @attention_f8(%query: tensor<192x1024x64xf8E4M3FNUZ>,
 // CHECK:   arith.mulf
 // CHECK:   arith.addf
 // CHECK:   linalg.yield
+
+// -----
+
+#mapQ = affine_map<(batch, m, k1, k2, n) -> (batch, m, k1)>
+#mapK = affine_map<(batch, m, k1, k2, n) -> (batch, k2, k1)>
+#mapV = affine_map<(batch, m, k1, k2, n) -> (batch, k2, n)>
+#mapS = affine_map<(batch, m, k1, k2, n) -> ()>
+#mapM = affine_map<(batch, m, k1, k2, n) -> (batch, m, k2)>
+#mapO = affine_map<(batch, m, k1, k2, n) -> (batch, m, n)>
+#mapR = affine_map<(batch, m, k1, k2, n) -> (batch, m)>
+
+func.func @attention_f8(%query: tensor<192x1024x64xf8E4M3FNUZ>,
```

Comment:
Please change the function name to reflect what it is testing. Also, I would prefer if this was testing fp16 attention. fp8 attention has other scaling that can be a red herring on failing tests.